lib/ui/window/platform_configuration.cc

```diff
@@ -424,6 +442,20 @@ void PlatformConfiguration::CompletePlatformMessageResponse(
   response->Complete(std::make_unique<fml::DataMapping>(std::move(data)));
 }
 
+void PlatformConfiguration::CompleteKeyDataResponse(uint64_t response_id, bool handled) {
+  if (!response_id) {
+    return;
+  }
+  auto it = pending_key_responses_.find(response_id);
+  FML_DCHECK(it != pending_key_responses_.end());
+  if (it == pending_key_responses_.end()) {
+    return;
+  }
+  auto callback = std::move(it->second);
```

Comment:
The type for this `auto` isn't obvious locally, so per style guide prefer using an explicit type.